src/buffer.ts

```diff
@@ -151,7 +151,7 @@ export function createReadBuffer(buf: BufferSource): ReadBuffer {
 
 		get(len: number): ArrayBuffer {
 			n += len;
-			return view.buffer.slice(n-len, n);
```

Comment:
Maybe we can introduce a constant at the beginning of `createReadBuffer` with the buffer's byte offset, so we can use this constant instead of the `byteOffset`? I know it's minimal, but it should reduce the output size a little bit, after name mangling.

Reply:
How about?
```suggestion
			return view.buffer.slice(view.buffer.byteOffset+n-len, view.buffer.byteOffset+n);
```

Reply:
or you can read the `byfeOffset` on the line before and use it twice...

Reply:
> or you can read the `byteOffset` on the line before and use it twice...

This sounds good to me :)

Reply:
```suggestion
			const offset = view.buffer.byteOffset;
			return view.buffer.slice(offset+n-len, offset+n);
```